frontend/src/components/containers/ContainerFilters.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {filter as filterObject} from "rambda";
+import {Collapse} from "antd";
+
+import {setFilter} from "../../modules/containers/actions";
+import FilterGroup from "../filters/filterGroup/FilterGroup";
+import {CONTAINER_FILTERS} from "../filters/descriptions";
+import { useAppDispatch, useAppSelector } from "../../hooks";
+import { FilterDescription, FilterSetting, FilterValue } from "../../models/paged_items";
+
+type ContainerFiltersProps = Pick<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'style'>
+
+const ContainerFilters = (rest: ContainerFiltersProps) => {
+  const filters = useAppSelector((state) => state.containers.filters as FilterSetting[])
+  const dispatch = useAppDispatch()
+
+  const onChangeFilter = (filter: FilterDescription, value: FilterValue) => {
+    dispatch(setFilter(filter.key, value))
+  }
```

Comment:
use some sort of debounce to fix abort abuse.

Reply:
DebouncedInput.jsx has a debounce example, and JumpBar.js too.